OracleFMWInfrastructure/README.md

```diff
@@ -72,7 +74,7 @@ If you need to find the passwords at a later time, grep for "password" in the Do
 ### Write your own Oracle Fusion Middleware Infrastructure domain with WLST
 The best way to create your own, or extend domains is by using [WebLogic Scripting Tool](https://docs.oracle.com/middleware/1221/cross/wlsttasks.htm). You can find an example of a WLST script to create domains at [createInfraDomain.py](dockerfiles/12.2.1.2/container-scripts/createInfraDomain.py). You may want to tune this script with your own setup to create DataSources and Connection pools, Security Realms, deploy artifacts, and so on. You can also extend images and override an existing domain, or create a new one with WLST.
```

Comment:
Rewrite to: "The best way to create your own domain or extend an existing domain is by using the WebLogic Scripting tool".